mockbee-test/ecom-store/src/backend/db/products.js

```diff
@@ -8,23 +8,105 @@ import { v4 as uuid } from "uuid";
 export const products = [
   {
     _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    imgUrl: "https://therightfit.netlify.app/assets/shoes.jpg",
+    isTrending: false,
+    isFavourite: false,
+    title: "Nike",
+    productCategory: "Men Sneakers",
+    newPrice: 1750,
+    oldPrice: 2499,
+    discount: Math.floor(Math.random() * 100)
```

Comment:
You can add `inStock` as well to product list

Reply:
I added inStock and Fast delivery both,